wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/cors/CorsConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package com.cognifide.aem.stubs.wiremock.cors;
+
+public class CorsConfiguration {
```

Comment:
instead of per feature objects maybe introduce `WiremockServletConfig` object in which we will manipulate all features - this CORS feature and all others in the future.

I don't want to have the situation in which we will have n objects passed to WiremockServlet obj constructor.

Reply:
that object `WiremockServletConfig` will be stateful builder-like object

Reply:
I added this part to existing class WiremockOprions, Thanks for comment :)